docs/install.rst

```diff
@@ -5,31 +5,77 @@ Installation of Cascade
 -----------------------
 Cascade interacts with Dismod-AT underneath. Cascade runs Dismod-AT within
 the IHME infrastructure. Clone it from
-`Cascade on Github <https://github.com/ihmeuw/cascade>`_. Then
-use ``python setup.py install`` to install it into the current
-Python environment or virtual environment.
+`Cascade on Github <https://github.com/ihmeuw/cascade>`_.
+We recommend you create a virtual environment into which to install
+the code. Then::
 
+    git clone https://github.com/ihmeuw/cascade.git
+    # Or use the one below if you have a Github account.
+    # git clone git@github.com:ihmeuw/cascade.git
+    cd cascade
+    pip install .[ihme_requirements,testing]
+    python setup.py develop
+    cd tests && pytest
+
+Then you should find that the current Python virtual environment
+has several new scripts installed.
 
 
 Installation of Dismod-AT
 -------------------------
+Dismod-AT is already installed on the cluster. If you've installed the
+Cascade as described above, it will have created two commands,
+``dmdismod`` and ``dmdismodpy``, which are the application and its
+Python helper. You can run, for instance::
+
+    dmdismod data.db init
+    dmdismod data.db fit
```

Comment:
(env) [nhawkins@cn497 cascade]$ dmdismod data.db fit
dismod_at-20180807 release build
fit command expected 1 or 2 arguments to follow fit